### PR TITLE
Connect marketplace API to microservice

### PR DESCRIPTION
**Pull request by reivaj05**
Adds a request to marketplace microservice to store email in DB

It is related to issue:
https://github.com/miniohq/engineering/issues/718

### CODE REVIEW

**Review by cniackz — DISMISSED** [review]
It looks good to me; I was learning from @reivaj05 how to test this new feature and it looks amazing 👍

**Review by dvaldivia — APPROVED** [review]

**Review by dvaldivia — APPROVED** [review]

**Review by dvaldivia — DISMISSED** [review]

**Review by bexsoft — APPROVED** [review]

File: operatorapi/marketplace.go [hunk 1]
```diff
@@ -19,22 +19,33 @@ package operatorapi
...
 	mpConfigMapDefault = "mp-config"
 	mpConfigMapKey     = "MP_CONFIG_KEY"
-	mpEmail            = "email"
+	mpHostEnvVar       = "MP_HOST"
+	defaultMPHost      = "https://marketplace.apps.min.dev"
+	mpEUHostEnvVar     = "MP_EU_HOST"
+	defaultEUMPHost    = "https://marketplace-eu.apps.min.dev"
+	isMPEmailSet       = "isEmailSet"
 	emailNotSetMsg     = "Email was not sent in request"
 )
```
Comment (harshavardhana) [inline operatorapi/marketplace.go:50]: All console ENVs must have a CONSOLE_ prefix

File: operatorapi/marketplace.go [hunk 2]
```diff
@@ -56,62 +67,112 @@ func registerMarketplaceHandlers(api *operations.OperatorAPI) {
...
+func makePostRequestToMP(url, email string) error {
+	request, err := createMPRequest(url, email)
+	if err != nil {
+		return err
+	}
+	client := &http.Client{Timeout: 3 * time.Second}
+	if res, err := client.Do(request); err != nil {
+		return err
+	} else if res.StatusCode >= http.StatusBadRequest {
+		b, _ := io.ReadAll(res.Body)
```
Comment (harshavardhana) [inline operatorapi/marketplace.go:154]: resp.Body is not closed so this is leaking connections  - repeated use of this function can slowly leak connections unknowingly.

File: operatorapi/marketplace.go (hunk 1 above)
Comment (harshavardhana) [reply operatorapi/marketplace.go:50]: We should avoid generic environment variables always.

**Pull request merged**